openpype/modules/sync_server/sync_server.py

```diff
@@ -229,6 +244,203 @@ def _get_configured_sites_from_setting(module, project_name, project_setting):
     return configured_sites
 
 
+def get_last_published_workfile_path(
```

Comment:
There is too much arguments in this function. Also I'm not sure if it should be a function?
At the end its just this
```
path = None
if workfile_representation:
    path = get_representation_path_with_anatomy(
        workfile_representation, anatomy
    )
```

Reply:
Done